src/lib.rs

```diff
@@ -526,13 +522,41 @@ impl PurpleICQ {
             None => {
                 let mut conversation = connection.serv_got_joined_chat(&info.sn).unwrap();
                 conversation.set_data("sn", &info.sn);
-                if let Some(stamp) = &info.stamp {
-                    conversation.set_data("stamp", stamp);
-                }
                 conversation.set_title(&info.title);
             }
         }
     }
+
+    pub fn load_chat_info(&mut self, connection: &mut Connection, info: &ChatInfo) {
+        log::debug!("loading chat info: {:?}", info);
+
+        let mut chat = self.chat_joined(
+            connection,
+            &info.as_partial(),
+            &ChatInfoVersionHandler::DoNothing,
+        );
+
+        match connection.get_account().find_chat_conversation(&info.sn) {
+            Some(mut conversation) => {
+                let mut node = chat.as_blist_node();
+                info.write_version_info(&mut node);
+
+                let mut chat_conversation = conversation.get_chat_data().unwrap();
+                chat_conversation.clear_users();
+                for member in &info.members {
+                    chat_conversation.add_user(
```

Comment:
What happens when I click on it? It seems that it opens a tab, will it be the right sn?